File: commons/src/main/java/com/eirsteir/coffeewithme/commons/domain/coffeebreak/CoffeeBreakDetails.java
package com.eirsteir.coffeewithme.commons.domain.coffeebreak;

import com.eirsteir.coffeewithme.commons.domain.university.CampusDetails;
import com.eirsteir.coffeewithme.commons.domain.user.UserDetails;
import java.time.LocalTime;
import java.util.Set;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class CoffeeBreakDetails {

  private LocalTime scheduledTo;
  private UserDetails requester;
  private Set<UserDetails> addressees;
  private CampusDetails campus;
}

File: commons/src/main/kotlin/com/eirsteir/coffeewithme/commons/domain/coffeebreak/CoffeeBreakDetails.kt
package com.eirsteir.coffeewithme.commons.domain.coffeebreak

import com.eirsteir.coffeewithme.commons.domain.university.CampusDetails
import com.eirsteir.coffeewithme.commons.domain.user.UserDetails
import java.time.LocalTime

data class CoffeeBreakDetails(
    val scheduledTo: LocalTime,
    val requester: UserDetails,
    val addressees: Set<UserDetails>,
    val campus: CampusDetails,
)